packages/gator-permissions-controller/package.json

```diff
@@ -49,6 +49,8 @@
   "dependencies": {
     "@metamask/7715-permission-types": "^0.3.0",
     "@metamask/base-controller": "^8.3.0",
+    "@metamask/delegation-core": "^0.2.0-rc.1",
```

Comment:
0.2.0 stable release is just around the corner, so we can probably update this before merging.